python/llm/src/bigdl/llm/optimize.py

```diff
@@ -133,6 +133,31 @@ def low_memory_init():
 
 
 def load_low_bit(model, model_path):
+    """
+    Load the optimized pytorch model.
+
+    :param model: The PyTorch model instance
+    :param model_path: The path of saved optimized model
+
+    return: The optimized model.
+
+    >>> Example 1:
+    >>> # Take OpenAI Whisper model as an example
+    >>> # Make sure you have saved the optimized model by calling 'save_low_bit'
+    >>> from bigdl.llm.optimize import load_low_bit
+    >>> model = whisper.load_model('tiny') # A model instance
+    >>> model = load_low_bit(model, saved_dir) # Load the optimized model
+
+    >>> Example 2:
+    >>> # Take ChatGLM2-6B model as an example
+    >>> # Make sure you have saved the optimized model by calling 'save_low_bit'
+    >>> from bigdl.llm.optimize import low_memory_init, load_low_bit
+    >>> with low_memory_init(): # Load model on meta device
+    >>>     model = AutoModel.from_pretrained(saved_dir,
+    >>>                                       torch_dtype="auto",
+    >>>                                       trust_remote_code=True)
+    >>> model = load_low_bit(model, saved_dir) # Load the optimized model
```

Comment:
I suppose maybe we should recommend users to try `low_memory_init()` first, what about make this the Example 1?

Reply:
> I suppose maybe we should recommend users to try `low_memory_init()` first, what about make this the Example 1?

yeah, I have changed the example order and updated related descriptions.